webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -5,15 +5,12 @@ import type {ReactNode} from 'react';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-import type {WarnMetricStatus} from '@mattermost/types/config';
-
 import {trackEvent} from 'actions/telemetry_actions.jsx';
 
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import OverlayTrigger from 'components/overlay_trigger';
 import ToggleModalButton from 'components/toggle_modal_button';
 import Tooltip from 'components/tooltip';
-import WarnMetricAckModal from 'components/warn_metric_ack_modal';
```

Comment:
@harshilsharma63  I was able to remove all code related to the `warn_metric_ack_modal` but in this AnnouncementBar seems to be still referencing it here but I don't see any condition being checked for warn metric to easily remove it. This is is the generic announcement bar so not sure  if I can safely remove it. Can you help on this? (see line here: https://github.com/mattermost/mattermost/blob/d3472e58119c44c3d3505b0890054093297f4675/webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx#L199 it doesn't let me comment there)

Reply:
@marianunez , might have to do with `showModal` which appears to be used only for admin_advisor-related stuff - see `configuration_bar.tsx`. I guess at the time the thinking was that this will be other instances of showing the modal besides the admin_advisor stuff, hence the generic name.